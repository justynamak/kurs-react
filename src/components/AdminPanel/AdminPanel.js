import React, { Component } from "react";

class AdminPanel extends Component {
  render() {
    return <div>Panel administratora</div>;
  }
}
export default AdminPanel;
