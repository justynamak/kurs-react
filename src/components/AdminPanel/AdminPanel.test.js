import React from "react";
import ReactDOM from "react-dom";
import AdminPanel from "./AdminPanel";
import Enzyme, { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("AdminPanel rendered", () => {
  it("AdminPanel rendered without problem", () => {
    const div = document.createElement("div");
    ReactDOM.render(<AdminPanel />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it("AdminPanel rendered", () => {
    const wrapper = shallow(<AdminPanel />);

    expect(wrapper.find("div").text()).toBe("Panel administratora");
  });
});
