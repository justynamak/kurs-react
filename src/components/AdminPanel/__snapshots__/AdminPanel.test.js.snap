// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminPanel rendered AdminPanel rendered 1`] = `ShallowWrapper {}`;
